### ISSUE TITLE
PandocUpload doesn't work on 1.39

### ISSUE THREAD

**Issue by Griboedow** [microsoft/MediaWiki-Extensions#2]
It calls doEditContent which is removed from 1.39.  https://phabricator.wikimedia.org/T286139

```
[91f73283841c3245fe4d337a] /index.php/Special:Upload Error: Call to undefined method WikiPage::doEditContent()

Backtrace:

from /etc/mediawiki/extensions/PandocUpload/PandocUpload.hooks.php(106)
#0 /etc/mediawiki/includes/HookContainer/HookContainer.php(338): PandocUploadHooks::onUploadComplete()
#1 /etc/mediawiki/includes/HookContainer/HookContainer.php(137): MediaWiki\HookContainer\HookContainer->callLegacyHook()
#2 /etc/mediawiki/includes/HookContainer/HookRunner.php(4027): MediaWiki\HookContainer\HookContainer->run()
#3 /etc/mediawiki/includes/upload/UploadBase.php(959): MediaWiki\HookContainer\HookRunner->onUploadComplete()
#4 /etc/mediawiki/includes/specials/SpecialUpload.php(614): UploadBase->performUpload()
#5 /etc/mediawiki/includes/specials/SpecialUpload.php(246): SpecialUpload->processUpload()
#6 /etc/mediawiki/includes/specialpage/SpecialPage.php(701): SpecialUpload->execute()
#7 /etc/mediawiki/includes/specialpage/SpecialPageFactory.php(1428): SpecialPage->run()
#8 /etc/mediawiki/includes/MediaWiki.php(316): MediaWiki\SpecialPage\SpecialPageFactory->executePath()
#9 /etc/mediawiki/includes/MediaWiki.php(904): MediaWiki->performRequest()
#10 /etc/mediawiki/includes/MediaWiki.php(562): MediaWiki->main()
#11 /etc/mediawiki/index.php(50): MediaWiki->run()
#12 /etc/mediawiki/index.php(46): wfIndexMain()
#13 {main}
``` 

You can replace it to doUserEditContent:
```
$page->doUserEditContent( $newContent, $user, "pandoc conversion");
```

**Comment by jeffw16** [comment]
Hi @Griboedow,

Thank you for filing this bug report. The extension was only updated to work on MediaWiki 1.35 at the time of open sourcing, so your assistance with providing 1.39 compatibility is much appreciated.

If you would be so kind as to fill out the CLA (instructions are available on this repo's README) and create a PR for this, we will be easily able to address this issue.

Thank you very much!

**Comment by jeffw16** [comment]
Fixed by #3
